keyboards/hecomi/alpha/config.h

```diff
@@ -0,0 +1,254 @@
+/*
```

Comment:
Most of this is identical to the config in `hecomi`, and this file could be removed.